### ISSUE TITLE
Running `containerbuild-regionsrv` throws `AttributeError` on SLES12sp5

### ISSUE THREAD

**Issue by bradfordb-vmware** [SUSE-Enceladus/cloud-regionsrv-client#36]
When running `containerbuild-regionsrv` (with systemd service or by hand) results in the following stacktrace:

```console
$ /usr/sbin/containerbuild-regionsrv
Traceback (most recent call last):
  File "/usr/sbin/containerbuild-regionsrv", line 109, in <module>
    main()
  File "/usr/sbin/containerbuild-regionsrv", line 104, in main
    with socketserver.TCPServer((ip, port), ContainerBuildTCPServer) as server:
AttributeError: __exit__
```

System info:

```console
$ cat /etc/os-release
NAME="SLES"
VERSION="12-SP5"
VERSION_ID="12.5"
PRETTY_NAME="SUSE Linux Enterprise Server 12 SP5"
ID="sles"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:12:sp5"

$ python3 --version
Python 3.4.10
```

**Comment by rhafer** [comment]
@bradfordb-vmware This should be addressed with #37 if you're interested in giving it a try.

**Comment by bradfordb-vmware** [comment]
> @bradfordb-vmware This should be addressed with #37 if you're interested in giving it a try.

Yep, that solves my issue. I am able to run the service on SLES12sp5 now. Thanks!
